packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -836,10 +851,16 @@ const Messages: FunctionComponent<MessagesProps> = ({
         onCloseDeletingErrorModal={hideDeleteMessageErrorModal}
         onDelete={handleDeleteMessage}
       />
-      {messagePopupNotifications.length > 0 && (
+      {showMessageDeletedModal && (
```

Comment:
Maybe we should unify names `showThreadDeletedPopup` and `showMessageDeletedModal`?

Reply:
Sure! [cdb4aea](https://github.com/mudita/mudita-center/pull/996/commits/cdb4aead3df31d3ade0b8b629cc868077614298c)